code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -907,7 +908,21 @@
 		for(var/obj/item/alien_embryo/embryo in potential_host)
 			embryo.hivenumber = XENO_HIVE_FORSAKEN
 		potential_host.update_med_icon()
+	var/shipside_humans_count = 0
+	var/xenos_count = 0
+	for(var/mob/living/carbon/human/current_human as anything in GLOB.alive_human_list)
+		if(isspecieshuman(current_human) || isspeciessynth(current_human))
+			var/atom/where = current_human
+			if (where.z == 0 && current_human.loc)
+				where = current_human.loc
+			if(is_mainship_level(where.z))
+				shipside_humans_count++
+	for(var/mob/living/carbon/xenomorph as anything in totalXenos)
```

Comment:
typing not valid

Reply:
Do you want me to rename totalXenos list?